### PR TITLE
Add forwarding of UIAccessibilityAction methods

### PR DESCRIPTION
**Pull request by smeis**
I was working on accessibility in our app and noticed these weren't implemented yet. There's no checking if the node implements the method because `NSObject` provides default implementations.

### CODE REVIEW

**Comment by smeis** [comment]
Hi @maicki @appleguy , I can't add reviewers but you've worked a lot on these components, could you take a look at it?

File: Tests/ASDisplayViewAccessibilityTests.mm [hunk 1]
```diff
@@ -269,4 +312,25 @@ - (void)testAccessibilityUpdatesWithElementsChanges
...
+  [view accessibilityIncrement];
+  XCTAssertTrue(node.didReceiveAccessibilityIncrement);
+  [view accessibilityDecrement];
+  XCTAssertTrue(node.didReceiveAccessibilityDecrement);
+  [view accessibilityScroll:UIAccessibilityScrollDirectionDown];
+  XCTAssertTrue(node.didReceiveAccessibilityScroll);
+  [view accessibilityPerformEscape];
+  XCTAssertTrue(node.didReceiveAccessibilityPerformEscape);
+  [view accessibilityPerformMagicTap];
+  XCTAssertTrue(node.didReceiveAccessibilityPerformMagicTap);
```
Comment (Adlai-Holler) [inline Tests/ASDisplayViewAccessibilityTests.mm]:
```suggestion
id mockNode = OCMPartialMock(node);
OCMExpect([node accessibilityActivate]);
[view accessibilityActivate];
OCMExpect([node accessibilityIncrement]);
[view accessibilityIncrement];
// …
OCMVerifyAll(mockNode);
```

Comment (smeis) [reply Tests/ASDisplayViewAccessibilityTests.mm]: @Adlai-Holler fixed!

**Comment by Adlai-Holler** [comment]
Thanks for the fixup @smeis , hope we see more of you!!

**Comment by fruitcoder** [comment]
Is there anything specific to watch out for? I tried subclassing an ASScrollNode and the overridden `accessibiliyActivate` isn't called

**Comment by maicki** [comment]
@fruitcoder As `ASScrollNode` uses `UIScrollView` and not `_ASDisplayView` as its view this methods will not be passed through to the node. We would have to add them to the `ASScrollView` class.

**Comment by fruitcoder** [comment]
I think I tried that as well but it didn't work since I can't provide a `ASScrollView` subclass that overrides the `accessibilityActivate` so I just added an a11y proxy node as the only scroll node's child when accessibility is enabled

**Comment by maicki** [comment]
You should be able to do this by subclassing `ASScrollView` and `ASScrollNode` and instead of creating the `ASScrollView` as view of the `ASScrollNode` within the initializer you are initialize your custom `ASScrollView`.  I'm not sure right now which one takes presedence but you also should be able to archive the same by overriding the `+[ASDisplayNode viewClass]` method

**Pull request merged**